src/Libraries/Microsoft.Extensions.Diagnostics.Testing/Microsoft.Extensions.Diagnostics.Testing.json

```diff
@@ -114,7 +114,7 @@
       ]
     },
     {
-      "Type": "class Microsoft.Extensions.Logging.Testing.FakeLogger : Microsoft.Extensions.Logging.ILogger, Microsoft.Extensions.Logging.Abstractions.IBufferedLogger",
+      "Type": "class Microsoft.Extensions.Logging.Testing.FakeLogger : Microsoft.Extensions.Logging.ILogger",
```

Comment:
I need someone to review this in particular. It seems that the baseline has this in, but we're targeting assemblies that do not have access to this interface?